db/migrations/20190711084519_create_flap_storage_tables.sql

```diff
@@ -157,9 +191,14 @@ CREATE TABLE maker.flap_kicks
 
 CREATE INDEX flap_kicks_header_id_index ON maker.flap_kicks (header_id);
 CREATE INDEX flap_kicks_address_index ON maker.flap_kicks (address_id);
+CREATE INDEX flap_kicks_kicks_index ON maker.flap_kicks (kicks);
+
+COMMENT ON TABLE maker.flap_kicks
+    IS E'@name flapKicksStorage\n@omit';
+
 
 -- prevent naming conflict with maker.flap_kick in postgraphile
-COMMENT ON TABLE maker.flap_kicks IS E'@name flapKicksStorage';
+-- COMMENT ON TABLE maker.flap_kicks IS E'@name flapKicksStorage';
```

Comment:
I wonder if we can delete this comment all together since we're including the renaming comment up on line 197.